test/everypolitician/popolo/collection_test.rb

```diff
@@ -64,4 +64,16 @@ def test_where_finding_by_multiple_attributes
     assert_equal popolo.organizations.where(classification: 'party', name: 'The Greens').count, 1
     assert_equal popolo.organizations.where(classification: 'party', name: 'The Greens').first.id, 'greens'
   end
+
+  def test_where_finding_some_attributes_with_no_matches
```

Comment:
I think I'd be more comfortable replacing all of this with a test that runs against something in `test/fixtures` rather than JSON that's manufactured just for the tests. We've hit quite a few bugs now where I think tests against real data might have highlighted the issues, or likely issues, much sooner.